protostar/commands/test/messages/test_case_result_message.py

```diff
@@ -0,0 +1,65 @@
+from typing import Any
+from dataclasses import dataclass
+from pathlib import Path
+from abc import abstractmethod
+
+from protostar.io import StructuredMessage, LogColorProvider
+from protostar.testing import (
+    OutputName,
+    format_output_name,
+)
+
+from protostar.starknet import ExceptionMetadata
+
+JsonData = dict[str, Any]
+
+
+@dataclass
+class TestCaseResultMessage(StructuredMessage):
+    @abstractmethod
+    def format_human(self, fmt: LogColorProvider) -> str:
+        assert False
+
+    @abstractmethod
+    def format_dict(self) -> dict:
+        assert False
```

Comment:
It looks like this class is no longer necessary.

Reply:
Ah, right, I added it because it would specify some fields that would be required in the children's classes but had to resign from that in the meantime, thanks for catching it!